ddl/ddl_api.go

```diff
@@ -3566,7 +3566,7 @@ func checkIsDroppableColumn(ctx sessionctx.Context, t table.Table, spec *ast.Alt
 	colName := spec.OldColumnName.Name
 	col := table.FindCol(t.VisibleCols(), colName.L)
 	if col == nil {
-		err = ErrCantDropFieldOrKey.GenWithStack("column %s doesn't exist", colName)
+		err = ErrCantDropFieldOrKey.GenWithStack("Can't DROP '%s'; check that column/key exists", colName)
```

Comment:
maybe `ErrCantDropFieldOrKey.GenWithStackByArgs(colName)` is ok?

Reply:
Nope~ according to #26137, we should keep the same error message with the mysql

Reply:
right, but you can see the template of `ErrCantDropFieldOrKey`, it is `Can't DROP '%-.192s'; check that column/key exists`. No need to write the template again.

btw, there are some other usages of `ErrCantDropFieldOrKey`, could please check them?

Reply:
I see~Thanks to your suggestion, I modified another place as well.